modules/eks/eks.tf

```diff
@@ -63,7 +64,7 @@ module "cluster" {
       root_volume_size              = lookup(node_pool, "volume_size")
       key_name                      = aws_key_pair.nodes.key_name
       public_ip                     = false
-      subnets                       = var.subnetworks
+      subnets                       = lookup(node_pool, "subnetworks")
```

Comment:
Missing the rename here

Reply:
nope, it's the key used by the official terraform module